tests/Helpers/ReferencedNameHelperTest.php

```diff
@@ -44,6 +44,29 @@ public function testGetAllReferencedNames()
 		}
 	}
 
+	/**
+	 * @requires PHP 7.1
+	 */
+	public function testGetAllReferencedNamesWithNullableTypehints()
```

Comment:
@pepakriz To `@requires` je zbytečné. Mělo by to fungovat i na PHP 7.0. Jediné, co nefunguje na PHP 7.0, je lint a to si vyřešil správně oddělením do souboru.